File: app/src/main/java/com/rafaeltmbr/stopwatch/domain/entities/Lap.kt
package com.rafaeltmbr.stopwatch.domain.entities


data class Lap(
    val index: Int,
    val milliseconds: Long = 0L,
    val status: Status = Status.CURRENT
) {
    enum class Status {
        CURRENT,
        BEST,
        WORST,
        DONE
    }
}

File: ios/Stopwatch/domain/entities/Lap.swift
enum LapStatus {
    case current, best, worst, done
}
struct Lap {
    let index: Int
    let milliseconds: Int
    let status: LapStatus
    
    init(index: Int, milliseconds: Int, status: LapStatus) {
        self.index = index
        self.milliseconds = milliseconds
        self.status = status
    }
}
